src/Models/IoTWorkspaceProject.ts

```diff
@@ -359,35 +346,38 @@ export class IoTWorkspaceProject extends IoTWorkbenchProjectBase {
   }
 
   /**
-   * Init Azure components and azure configs.
+   * Load project config and init Azure components.
    * @param scaffoldType scaffold type
    */
-  private async initAzureConfig(scaffoldType: ScaffoldType): Promise<void> {
-    this.validateProjectRootPath("initialize azure configurations", scaffoldType);
-
+  private async loadAzureConfigAndInitAzureComponents(scaffoldType: ScaffoldType): Promise<void> {

```

Comment:
What about simplify the name to "loadAzureConfigAndComponents"?

Reply:
Fixed.